spec/index.bs

```diff
@@ -2719,6 +2719,16 @@ enum GPUTextureAspect {
             </dl>
 </div>
 
+<div algorithm>
+    Two {{GPUTextureView}} objects |a| and |b| are considered <dfn dfn>texture-view-aliasing</dfn> if and only if all of the following are true:
+        - |a|.{{GPUTextureView/[[texture]]}} == |b|.{{GPUTextureView/[[texture]]}}.
+        - Let {{GPUTextureViewDescriptor}} |aDescriptor| = |a|.{{GPUTextureView/[[descriptor]]}}.
+        - Let {{GPUTextureViewDescriptor}} |bDescriptor| = |b|.{{GPUTextureView/[[descriptor]]}}.
+        - |aDescriptor|.{{GPUTextureViewDescriptor/aspect}} == {{GPUTextureAspect/"all"}}, or |bDescriptor|.{{GPUTextureViewDescriptor/aspect}} == {{GPUTextureAspect/"all"}}, or |aDescriptor|.{{GPUTextureViewDescriptor/aspect}} == |bDescriptor|.{{GPUTextureViewDescriptor/aspect}}.
+        - The range formed by |aDescriptor|.{{GPUTextureViewDescriptor/baseMipLevel}} and |aDescriptor|.{{GPUTextureViewDescriptor/mipLevelCount}} intersects the range formed by |bDescriptor|.{{GPUTextureViewDescriptor/baseMipLevel}} and |bDescriptor|.{{GPUTextureViewDescriptor/mipLevelCount}}.
+        - The range formed by |aDescriptor|.{{GPUTextureViewDescriptor/baseArrayLayer}} and |aDescriptor|.{{GPUTextureViewDescriptor/arrayLayerCount}} intersects the range formed by |bDescriptor|.{{GPUTextureViewDescriptor/baseArrayLayer}} and |bDescriptor|.{{GPUTextureViewDescriptor/arrayLayerCount}}.
```

Comment:
Just to confirm: buffer aliasing cares occurs only on overlapping ranges within the buffer, but texture aliasing occurs when the set of subresources overlaps at all?

(This is fine, just double checking it was intended)

Reply:
I didn't think the `renderExtent` field was relevant, because the prose below its definition says:

> For renderable views

And the [compatible usage list](https://gpuweb.github.io/gpuweb/#compatible-usage-list) section describes that if a view has the [`attachment`](https://gpuweb.github.io/gpuweb/#internal-usage-attachment) internal usage, it can't have the [`storage`](https://gpuweb.github.io/gpuweb/#internal-usage-storage) internal usage.

Am I misunderstanding something?

Reply:
Sorry, I didn't think about this hard enough. Indeed texture views can only refer to sets of whole subresources, not to subregions of resources. I forgot that the binding itself can refer to a region of a buffer. I guess my question should have been whether what matters is the _binding_, or the actual access pattern of the program, but we already decided that we would add language about the binding, not the access pattern. So this comment didn't make sense.

(BTW I think that's just a convenience for the validation steps in render passes.)